tamr_unify_client/models/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,79 @@
+        """Retrieve an attribute mapping by relative ID.
+       :param relative_id: The relative ID.
+       :type relative_id: str
+       :returns: The specified attribute mapping.
+       :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+       """
+        return super().by_relative_id(AttributeMapping, relative_id)
+
+    def by_external_id(self, external_id):
+        """Retrieve an attribute mapping by external ID.
+        Since attributes do not have external IDs, this method is not supported and will
+        raise a :class:`NotImplementedError` .
+        :param external_id: The external ID.
+        :type external_id: str
+        :returns: The specified attribute mapping, if found.
+        :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+        :raises KeyError: If no attribute mapping with the specified external_id is found
+        :raises LookupError: If multiple attribute mappings with the specified external_id are found
+        :raises NotImplementedError: AttributeMapping does not support external_id
+        """
+        raise NotImplementedError("AttributeMapping does not support external_id")
+
+    def by_resource_id(self, resource_id):
+        """Retrieve an attribute mapping by resource ID.
+        Since attributes do not have resource IDs, this method is not supported and will
+        raise a :class:`NotImplementedError` .
+        :param resource_id: The resource ID.
+        :type resource_id: str
+        :returns: The specified attribute mapping, if found.
+        :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+        :raises KeyError: If no attribute mapping with the specified resource_id is found
+        :raises LookupError: If multiple attribute mappings with the specified resource_id are found
+        :raises NotImplementedError: AttributeMapping does not support resource_id
+        """
+        raise NotImplementedError("AttributeMapping does not support resource_id")
+
+    def stream(self):
+        """Stream attribute mappings in this collection. Implicitly called when iterating
+         over this collection.
+         :returns: Stream of attribute mappings.
+         :rtype: Python generator yielding :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+         Usage:
+             >>> for attributeMapping in attribute_mapping__collection.stream(): # explicit
+             >>>     do_stuff(attributeMapping)
+             >>> for attributeMapping in attribute_mapping__collection: # implicit
+             >>>     do_stuff(attributeMapping)
+         """
+
+        return super().stream(AttributeMapping)
+
+    def create(self, creation_spec):
+        """Create an Attribute mapping in this collection
+       :param creation_spec: Attribute mapping creation specification should be formatted as specified in the
+<<<<<<< Updated upstream
```

Comment:
@charlottemoremen looks like there's a remnant of an unresolved `git` conflict here. Make sure to fix this before PR gets merged!